src/Features/Core/Portable/Features.csproj

```diff
@@ -109,5 +109,8 @@
   <ItemGroup>
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="RemoveUnusedMembers\" />
```

Comment:
Seems odd.

Reply:
Hrmm.. strange.  That was work in another branch.  i wonder how that happened.  Will address.

Reply:
Fixed.